index.js

```diff
@@ -309,15 +316,15 @@
 
       return bbox
     }
-    
+
     // Private - replace D3JS 3.X d3.functor() function
     function functor(v) {
-    	return typeof v === "function" ? v : function() {
+      return typeof v === 'function' ? v : function() {
         return v
-    	}
+      }
     }
 
     return tip
-  };
-
+  }
+// eslint-disable-next-line semi
```

Comment:
Actually now that I look at it, the header comments should act as a buffer for the top. Any theoretical concatenation library that would strip comments but not insert semicolons would be broken by design.